app/src/main/java/com/android/voyageur/model/activity/Activity.kt

```diff
@@ -4,13 +4,13 @@ import com.android.voyageur.model.location.Location
 import com.google.firebase.Timestamp
 
 data class Activity(
-    val title: String,
+    val title: String = "",
     val description: String = "",
-    val location: Location = Location(),
-    val startTime: Timestamp = Timestamp(0, 0),
-    val endDate: Timestamp = Timestamp(0, 0),
-    val estimatedPrice: Number = 0.0,
-    val activityType: ActivityType = ActivityType.OTHER,
+    val location: Location = Location(""),
+    val startTime: Timestamp = Timestamp.now(),
+    val endTime: Timestamp = Timestamp.now(),
```

Comment:
Drafts have the property of having Timestamp(0, 0), please change the default value to Timestamp(0,0).